tests/garage/torch/policies/test_gaussian_mlp_policy.py

```diff
@@ -0,0 +1,104 @@
+import pickle
+from unittest import mock
+
+import numpy as np
+import pytest
+import torch
+
+from garage.torch.policies import GaussianMLPPolicy
+
+
+test_settings = [
+    (1, 1, (1,)),
+    (1, 2, (2,)),
+    (1, 3, (3,)),
+    (1, 1, (1, 2)),
+    (1, 2, (2, 1)),
+    (1, 3, (4, 5)),
+    (2, 1, (1,)),
+    (2, 2, (2,)),
+    (2, 3, (3,)),
+    (2, 1, (1, 2)),
+    (2, 2, (2, 1)),
```

Comment:
be careful about how long the test takes to run.

Reply:
on my local machine, it was just taking a few seconds.
so I think it would be fine